src/utils/index.js

```diff
@@ -79,9 +79,13 @@ var reparentActor = (actor, parent) => {
     if (currentParent) {
         currentParent.remove_child(actor);
     }
-    parent.add_child(actor);
-    if (isFocused) {
-        actor.grab_key_focus();
+    try { // FIXME: workaround from crash on disabling = status area gone
+        parent.add_child(actor);
+        if (isFocused) {
+            actor.grab_key_focus();
+        }
+    } catch (error) {
+        logError(error, '[material-shell.index]');
```

Comment:
This tend to hide the poop instead of fixing the source and in this case I think I found the real source issue !

Reply:
Yeah, I know. That is why I added the FIXME tag: temporary workaround (avoid worse things) until properly fixed.

Reply:
Did you check my last commit for the crash?

Reply:
This error is always seen right after the one prevented by the try/catch block above, maybe it helps you fixing the cause of the problem:

```
set 26 20:03:17 miix320-pop gnome-shell[70073]: JS ERROR: Exception in callback for signal: tileable-focus-changed: Error: Argument 'child' (type interface) may not be null
                                                setTranslation/<@/home/lfom/.local/share/gnome-shell/extensions/material-shell@papyelgringo/src/widget/translationAnimator.js:92:50
```